### DISCUSSION TITLE
新的稍后再看页面 添加了”快速收藏 “功能了吗？

### DISCUSSION THREAD

**Discussion by kethoven** [the1812/Bilibili-Evolved#1666] [提问 / Q&A] [answered]
看最新版本的描述：
对新的稍后再看页面做一些兼容: 夜间模式 / 快速收藏 / 快捷键扩展等.
但是用了一下，没找到啊。
新的稍后再看页面 添加了”快速收藏 “功能了吗？

**Comment by the1812** [comment, marked as answer]
是稍后再看页面才有: `https://www.bilibili.com/medialist/play/watchlater/BVxxxxxxxx`
![image](https://user-images.githubusercontent.com/26504152/111773359-08b88500-88e9-11eb-88ea-740f2eaff0fe.png)

**Reply by kethoven** [reply]
请问是哪个版本啊？我的 火狐+976.80版本，总是找不到这个图标。

**Reply by the1812** [reply]
试了下火狐还真没有...
有点 bug

**Reply by kethoven** [reply]
原来是bug啊

**Reply by kethoven** [reply]
> 
> 
> 试了下火狐还真没有...
> 有点 bug
这个bug可以修复吗？

**Reply by the1812** [reply]
v1.12.3修了的
